scss/_settings_spacing.scss

```diff
@@ -147,6 +147,9 @@ $icon-sizes: (
   thumb--large: $sp-unit * 12
 ) !default;
 
+$table-cell-padding-top: $spv-inner--x-small--scaleable !default;
+$table-cell-padding-bottom: $spv-inner--x-small--scaleable !default;
```

Comment:
Wouldn't we want these to always be the same value? If these are override able (!default) then they should be documented?

Reply:
Good point. The only purpose of these is to ensure table cell padding and the overlap utility always match, so no need for `!default` and no need for separate top/bottom vars. I've updated the code.